Core/HLE/sceFont.cpp

```diff
@@ -35,9 +35,15 @@ enum {
 	FONT_IS_OPEN   = 1,
 };
 
+// For the save states.
+static bool useAllocCallbacks = true;
```

Comment:
We also need to reset it to `true` in `__FontInit()`.  This way, if you load an old save state, and then reset, we'll start using them again.

-[Unknown]